README.md

```diff
@@ -18,79 +20,140 @@ Rather than installing and configuring many different build tools, such as Webpa
+| [**clean-webpack-plugin**][10] | Cleans out the `dist` folder before starting a new build. |
+| [**css-loader**][11] | A Webpack loader which allows us to load CSS files with `@import` and `url()`. |
+| [**mini-css-extract-plugin**][12] | Webpack's architecture is built around JavaScript. Support has been added for CSS, however it results in CSS being embedded inside JavaScript files. This package allows us to export CSS into standalone files. |
+| [**sass-loader**][13] | The integration between Sass and Webpack. |
+| [**style-loader**][14] | Allows us to embed CSS into JavaScript. This is useful for functional CSS that is specifically related to a JavaScript module, for example the `.shade-bg` rule in [DD Shade](https://hub.deloittedigital.com.au/stash/projects/FED/repos/dd-shade/browse) |
+| [**webpack-cli**][15] | The command line interface for Webpack allows us to enter Webpack commands into our project's `package.json` file. |
+| [**webpack-serve**][16] | A lean, modern, and flexible Webpack development server which supports live reloading. |
+
+[7]: https://www.npmjs.com/package/autoprefixer
+[8]: https://www.npmjs.com/package/babel-loader
+[9]: https://www.npmjs.com/package/babel-preset-env
+[10]: https://www.npmjs.com/package/clean-webpack-plugin
+[11]: https://www.npmjs.com/package/css-loader
+[12]: https://www.npmjs.com/package/mini-css-extract-plugin
+[13]: https://www.npmjs.com/package/sass-loader
+[14]: https://www.npmjs.com/package/style-loader
+[15]: https://www.npmjs.com/package/webpack-cli
+[16]: https://www.npmjs.com/package/webpack-serve
 
 ### Vue.js packages
 
-| Name                            | Description                                                                                                                                                                                                                                                |
-|---------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **vue-loader**                  | A Webpack loader which allows us to use `*.vue` files.                                                                                                                                                                                                     |
-| **vue-template-compiler**       | Used to pre-compile Vue templates into render functions to avoid runtime-compilation overhead and CSP restrictions.                                                                                                                                        |
+| Name								| Description													|
```

Comment:
The first two tables have been updated to have a short header, i.e.
```
+| Name | Description |
+|------|-------------|
```
For consistency can you do this to the second two tables as well?

Reply:
Yeah I left the other two formatted because they fit inside the the 120 limit. But agree with the consistency thing. Have now updated.